src/shared/components/TagBadge.jsx

```
@@ -0,0 +1,14 @@
+import { Badge } from "@/components/ui/badge";
+import { cn } from "@/lib/utils";
+export const TagBadge = ({ tag, onClick, rightSlot, className }) => {
+  return (
+    <Badge
+      key={tag.id}
+      className={cn("m-1 bg-amber-500 font-bold", className)}
+      onClick={() => onClick && onClick()}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)